packages/snaps-controllers/src/test-utils/controller.ts

```diff
@@ -796,6 +800,33 @@ export const getRestrictedSnapInterfaceControllerMessenger = (
       result: false,
       type: 'all',
     }));
+
+    messenger.registerActionHandler(
+      'AccountsController:getAccountByAddress',
+      (address: string) =>
+        ({
+          address,
+          id: 'foo',
+          scopes: ['eip155:0'],
+        } as unknown as InternalAccount),
```

Comment:
If this is a partial mock, we should prefer `@ts-expect-error`. Same in other places below.